dashboards-observability/public/components/application_analytics/components/configuration.tsx

```diff
@@ -0,0 +1,158 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import { 
+    EuiButton, 
+    EuiFlexGroup, 
+    EuiFlexItem, 
+    EuiHorizontalRule, 
+    EuiInMemoryTable, 
+    EuiPage, 
+    EuiPageBody, 
+    EuiPageContent, 
+    EuiPageContentHeader, 
+    EuiPageContentHeaderSection, 
+    EuiTableFieldDataColumnType, 
+    EuiText, 
+    EuiTitle 
+} from '@elastic/eui';
+import React from 'react';
+
+const dummy = [{
+    level: "Available",
+    definition: "error rate below or equal to 1%",
+    id: "1"
+}];
+
+const dummyLogSources = [
+    {logName: "index_1"}, {logName: "ingest_logs_all"}
+];
+
+const dummyServicesEntities = [
+    {serviceName: "Payment"}, {serviceName: "Users"}, {serviceName: "Purchase"}
+];
+
+const dummyTraceGroups = [
+    {traceGroup: "Payment.auto"}, {traceGroup: "Users.admin"}, {traceGroup: "Purchase.source"}
+];
+
+export const Configuration = () => {
+
+    const tableColumns = [
+        {
+          field: 'level',
+          name: 'Level',
+          render: (value) => value,
+        },
+        {
+          field: 'definition',
+          name: 'Definition',
+          render: (value) => value,
+        },
+      ] as Array<
+        EuiTableFieldDataColumnType<{
+            level: string;
+            id: string;
+            definition: string;
+        }>
+      >;
+
+    return (
```

Comment:
Is this using 4-space indentation vs 2-space?

Reply:
It's using 4-space. It defaulted to that for some of my files and it was too much of a bother to fix it every time. Should I change it to 2-space? There is probably a way to change it in settings.

Reply:
Just changed it!